src/Vendoring/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This copyright is different. Intentional?

Reply:
Hmm, this file name says "SqlClient", doesn't seem to be related with what the PR title suggested.

Reply:
From previous base branch which got merged.